### DISCUSSION TITLE
使用安装脚本时遇到的问题

### DISCUSSION THREAD

**Discussion by iliaoke** [nikkinikki-org/OpenWrt-nikki#268] [General]
更新完软件源之后一直卡在download tarball

**Comment by morytyann** [comment]
大概网络不好，不行就手动下载吧

**Reply by mqcycy** [reply]
是否可以考虑增加延迟启动选项，发现重启openwrt后mihomo启动失败，查看日志发现下载订阅配置失败造成插件启动失败，点击重载即可成功启动，多次测试结论是上游WAN口网络未就绪但插件已启动，希望排查采纳，谢谢

**Reply by morytyann** [reply]
嗯，有计划添加
